autoload/sj/elixir.vim

```diff
@@ -39,3 +39,49 @@ function! sj#elixir#JoinDef()
   call sj#ReplaceLines(def_lineno, end_lineno, joined_line)
   return 1
 endfunction
+
+function! sj#elixir#SplitArray()
+  let [from, to] = sj#LocateBracesAroundCursor('[', ']', [
+        \ 'elixirInterpolationDelimiter',
+        \ 'elixirString',
+        \ 'elixirStringDelimiter',
+        \ 'elixirSigilDelimiter',
+        \ ])
+
+  if from < 0
+    return 0
+  endif
+
+  let items = sj#ParseJsonObjectBody(from + 1, to - 1)
+
+  " substitute [1, 2, | tail]
+  let items[-1] = substitute(items[-1], "\\(|[^>].*\\)", "\n\\1", "")
```

Comment:
We should check if `len(items) > 0` here -- if you try to split an empty array with a space in it (`[ ]`), this line blows up.

Reply:
Oh my, can't believe I didn't test that case 🤦